psi4/driver/procrouting/response/scf_response.py

```diff
@@ -780,126 +780,84 @@ def tda_solver(e, n, g, m):
             names.add("DFT")
 
         for name in names:
+
+            def root_setter(prop_name, val):
+                wfn.set_variable(f"TD-{name} ROOT {i+1} {prop_name}", val)
+                wfn.set_variable(f"TD-{name} ROOT {target_h_count} IN {x.irrep_ES} {prop_name}", val)
+                wfn.set_variable(f"TD-{name} ROOT {i+1} ({x.irrep_ES}) {prop_name}", val)
+                wfn.set_variable(f"TD-{name} ROOT {i+1} {prop_name} - {x.irrep_trans} TRANSITION", val)
+
+            def transition_setter(prop_name, val)
+                wfn.set_variable(f"TD-{name} ROOT 0 -> ROOT {i+1} {prop_name}", val)
+                wfn.set_variable(f"TD-{name} ROOT 0 IN {x.irrep_GS} -> ROOT {target_h_count} IN {x.irrep_ES} {prop_name}", val)
```

Comment:
I like the `IN` syntax, and I think this quartet covers the needs nicely. I've thought of a couple more variations that avoid me reading `i` as an excitation energy, but I'm not convinced either is better. good to go, as-is, imo.
```
ADC ROOT 0 IN h -> ROOT mm IN i EXCITATION ENERGY  # present

ADC ROOT 0 (IN h) -> ROOT mm (IN i) EXCITATION ENERGY  # alt1
ADC ROOT 0-IN-h -> ROOT mm-IN-i EXCITATION ENERGY  # alt2
```

Reply:
I'm favoring alt1, so that irrep information is now always in parentheses. Will get on this... later today (but at a reasonable time) or tomorrow.